### ISSUE TITLE
Compatibility request.

### ISSUE THREAD

**Issue by Timtee** [shaksternano/WMITAF#2]
I provide this because of one reason: I normally use Just Enough Items instead of REI. And well... I'd like to ask if it's possible that you could provide support for JEI on the forge build in particular. Reason being is that JEI and REI do not play nicely with one another and cause the game to _instantly_ crash as soon it even detects the mods. (So while the issue isn't from this mod directly, the requirement of a mod that conflicts with another that i prefer over the required one, is still a form of issue at the least.)

Also this is mostly for 1.18.2 as FYI I should mention.

**Comment by AmyRose1288** [comment]
indeed,pls make a JEI Compatible version (also,Jade is used way more than WTHIT/TOP)

**Comment by shaksternano** [comment]
JEI of Forge already does what this mod does for REI though?

**Comment by shaksternano** [comment]
I can do JEI and Jade support for Fabric though
